src/service/video-manager-impl.js

```diff
@@ -67,9 +67,9 @@ export class VideoManager {
 
     // TODO(aghassemi): Remove this later. For now, VideoManager only matters
     // for autoplay videos so no point in registering arbitrary videos yet.
-    if (!video.element.hasAttribute(VideoAttributes.AUTOPLAY)) {
-      return;
-    }
+    // if (!video.element.hasAttribute(VideoAttributes.AUTOPLAY)) {
```

Comment:
Wouldn't we need the visibility observer for docking videos whether autoplaying or not?